INFO     root:conftest.py:202 === Starting test_get_gas_fee Test ===
INFO     root:conftest.py:203 API Response: 500 | {"title":"A problem occurred processing your request","detail":"System.Exception: The address 0xE44872C526B025c7846Af34B2111F98c0483Bf8C balance of IZE is zero\n   at Nerd.Wallet.Core.Services.ERC20WalletService.GetGasLimit(ERC20Configuration erc20Configuration, String from, String to, BigInteger numOfTokens) in /src/Nerd.Wallet/Nerd.Wallet.Core/Services/ERC20WalletService.cs:line 181\n   at Nerd.Wallet.Core.Services.ERC20WalletService.GetGasFee(FeeForSendErc20Transaction request, Boolean core, CancellationToken cancellationToken) in /src/Nerd.Wallet/Nerd.Wallet.Core/Services/ERC20WalletService.cs:line 170\n   at Nerd.Wallet.Api.Controllers.V1.TransactionsController.GetGasFee(String symbol, FeeForSendTransactionRequest request, Boolean core, CancellationToken cancellationToken) in /src/Nerd.Wallet/Nerd.Wallet.Api/Controllers/V1/TransactionsController.cs:line 187\n   at lambda_method3697(Closure, Object)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(ActionContext actionContext, IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.\u003CInvokeActionMethodAsync\u003Eg__Awaited|12_0(ControllerActionInvoker invoker, ValueTask\u00601 actionResultValueTask)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.\u003CInvokeNextActionFilterAsync\u003Eg__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State\u0026 next, Scope\u0026 scope, Object\u0026 state, Boolean\u0026 isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.\u003CInvokeInnerFilterAsync\u003Eg__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.\u003CInvokeFilterPipelineAsync\u003Eg__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.\u003CInvokeAsync\u003Eg__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.\u003CInvokeAsync\u003Eg__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)\n   at Nerd.Common.Logging.NerdApiGlobalLogScopeMiddleware.InvokeAsync(HttpContext httpContext)\n   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\n   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddlewareImpl.\u003CInvoke\u003Eg__Awaited|10_0(ExceptionHandlerMiddlewareImpl middleware, HttpContext context, Task task)","ErrorCode":null,"ErrorDescription":null,"ErrorDetail":null,"Url":"http://api.dev.nerdunited.net/api/transactions/fee/ize?core=true"}
INFO     root:conftest.py:209 Test 'test_get_gas_fee' FAILED.
ERROR    root:conftest.py:210 API response status code: 500
assert 500 in [200, 201, 202, 203, 204, 205]
 +  where 500 = <Response [500]>.status_code
Traceback (most recent call last):
  File "/home/runner/work/qa-nerd-core-backend/qa-nerd-core-backend/conftest.py", line 207, in api_response_conditions
    assert response.status_code in allowed_status_codes, f"API response status code: {response.status_code}"
AssertionError: API response status code: 500
assert 500 in [200, 201, 202, 203, 204, 205]
 +  where 500 = <Response [500]>.status_code